org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/target/editor/ClipboardParser.java

```diff
@@ -70,16 +57,15 @@ private void parseElement(Element element) {
 		String version = getTextFor("version", element, "");
 		String classifier = getTextFor("classifier", element, "");
 		String type = getTextFor("type", element, MavenTargetLocation.DEFAULT_PACKAGE_TYPE);
-		this.dependencies
-				.add(new MavenTargetDependency(groupId, artifactId, version, type, classifier));
+		this.dependencies.add(new MavenTargetDependency(groupId, artifactId, version, type, classifier));
 	}
 
 	private String getTextFor(String element, Element doc, String defaultValue) {
 		NodeList nl = doc.getElementsByTagName(element);
 		Node item = nl.item(0);
 		if (item != null) {
-			String v = Objects.requireNonNullElse(item.getTextContent(), defaultValue);
-			if (!v.isBlank()) {
+			String v = item.getTextContent();
+			if (v != null && !v.isBlank()) {
```

Comment:
this do no longer return the defaultvalue when the text content is missing...

Reply:
Isn't that happening below in line 72?